librad/Cargo.toml

```diff
@@ -5,6 +5,10 @@ authors = ["The Radicle Team <dev@radicle.xyz>"]
 edition = "2018"
 license = "GPL-3.0-or-later"
 
+[features]
+default = []
+disco-mdns = ["mdns", "madness"]
```

Comment:
Seems like the right way to introducing capabilities like this. :+1:

Reply:
I feel like we missed an opportunity to call it: `disco-madness` :man_dancing:

Reply:
How would you feel about renaming `master` to `madness`?

Reply:
I'd feel good about it.

Reply:
> How would you feel about renaming master to madness?

➕